packages/block-library/src/navigation-link/edit.js

```diff
@@ -53,61 +54,61 @@ const DEFAULT_BLOCK = { name: 'core/navigation-link' };
  *
  * @return {boolean} Is dragging within the target element.
  */
-const useIsDraggingWithin = ( elementRef ) => {
-	const [ isDraggingWithin, setIsDraggingWithin ] = useState( false );
+const useIsDraggingWithin = (elementRef) => {
+	const [isDraggingWithin, setIsDraggingWithin] = useState(false);
 
-	useEffect( () => {
+	useEffect(() => {
 		const { ownerDocument } = elementRef.current;
 
-		function handleDragStart( event ) {
+		function handleDragStart(event) {
 			// Check the first time when the dragging starts.
-			handleDragEnter( event );
+			handleDragEnter(event);
```

Comment:
Could we please remove all the whitespace changes in this PR? 🙏 

It would make it a lot easier to review. Many thanks.

Reply:
@rinkalpagdar, unfortunately, [linting errors](https://github.com/WordPress/gutenberg/actions/runs/12352367237/job/34469254137?pr=67973) remain. I would recommend setting up [ESLint + Prettier](https://developer.wordpress.org/block-editor/contributors/code/getting-started-with-code-contribution/#developer-tools) and letting the editor handle stylistic fixes for you.

Reply:
These now look fixed to me thank you.